### PR TITLE
Consensus transaction list feedback

### PR DESCRIPTION
**Pull request by buberdds**
option with using `getNameForScope`

### CODE REVIEW

**Comment by buberdds** [comment]
https://github.com/oasisprotocol/towncrier/pull/9 :shrug:

File: src/app/utils/helpers.ts [hunk 1]
```diff
@@ -128,3 +129,11 @@ export const getAccountSize = (value: bigint) => {
     return 'XXS'
   }
 }
+
+export const getLayerByRuntimeId = (runtimeId: string) => {
+  return (Object.keys(paraTimesConfig) as Layer[]).find(
+    layer =>
+      paraTimesConfig[layer]?.mainnet.runtimeId === runtimeId ||
+      paraTimesConfig[layer]?.testnet.runtimeId === runtimeId,
+  )
```
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/utils/helpers.ts:138]: I would rather be network-specific, despite not printing the name if testnet roothashExecutorCommit somehow references a mainnet runtime id

File: src/app/components/Transactions/ConsensusTransactionDetails.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ const getConsensusTransactionDetails = (t: TFunction, transaction: Transaction,
           <From address={transaction.sender} ownAddress={ownAddress} scope={scope} />
           <To
             address={transaction.to}
-            label={t('validator.title')}
+            label={t('common.toValidator')}
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/components/Transactions/ConsensusTransactionDetails.tsx]: does `Start Undelegate;  From validator oasis1…095l5z; To oasis1…dtg2km; Shares 887M`  work for reclaim?

[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/components/Transactions/ConsensusTransactionDetails.tsx]: after product meeting will close PR for now as we need to wait for new designs

**Pull request closed without merging**